bundles/org.eclipse.jface/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %pluginName
 Bundle-SymbolicName: org.eclipse.jface;singleton:=true
-Bundle-Version: 3.36.100.qualifier
+Bundle-Version: 3.37.0.qualifier
```

Comment:
Somehow API-tools considers the removal of the (deprecated) overriden method from `CompositeImageDescriptor` as API addition and demands to minor version bump.